targets/drive/mobile/fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,11 +1,14 @@
-This application is only compatible with the latest version of Cozy (v3).
+We all have a life and a digital life. Cozy gathers them. In your Cozy, you have all your personal data: pictures, health, bank, holidays, accessible only by you.
+Here comes the Cozy first mobile app, in its Beta version (replacing the old V2 version): Cozy Drive.
 
-Cozy Cloud simplifies the personal cloud. Drive and Photos you keep in your Cozy, your personnal cloud, are accessible from everywhere.
+Cozy Drive Features:
+• Save all the photos on your smartphone, they will be available on your PC and tablet as well
+• Automatically synchronize the files on your PC with Cozy
+• Share your holiday pictures with everyone you want!
+• Compatible with MacOs, iOS, Android and Windows.
```

Comment:
??? What is compatible with macOS, iOS, Android and Windows?

Reply:
Cozy Drive. It's not clear indeed.